src/logger.ts

```diff
@@ -53,8 +54,8 @@ function redactMessage(message: string | unknown, options?: RuntimeConfig) {
 	const runtimeConfig = options ?? getRuntimeConfig();
 	let ret = message;
 
-	// redact torznab api keys
-	ret = ret.replace(/apikey=[a-zA-Z0-9]+/g, `apikey=${redactionMsg}`);
+	ret = ret.replace(/key=[a-zA-Z0-9]+/g, `key=${redactionMsg}`);
+	ret = ret.replace(/pass=[a-zA-Z0-9]+/g, `pass=${redactionMsg}`);
```

Comment:
```suggestion
	ret = ret.replace(/pass=[a-zA-Z0-9]+/g, `pass=${redactionMsg}`);
	ret = ret.replace(/(?:auto\.\d+\.)([a-zA-Z0-9]+)/g, (match, key) =>
		match.replace(key, redactionMsg),
	);
```
needed for a specific tracker's custom formatting. tested already.

Reply:
what format does it take?

Reply:
it appends the rss key to the URL in the format

tracker.com/download/auto.torrentid#.rsskey